src/platforms/platforms.ts

```diff
@@ -36,8 +45,16 @@ type PlatformData = {
   }
   cardSettingsComponent?: () => JSX.Element
   cardMenuComponent?: (props) => JSX.Element
-  cardWarningComponent?: (props) => JSX.Element
+  cardWarningComponent?: (props) => ReactNode
```

Comment:
I wonder if this change is needed? 👀

Reply:
Nvm, we talked about it on Discord, I guess we can revert this change, haha!

Reply:
Jeje, thanks, reverted in https://github.com/agoraxyz/guild.xyz/pull/858/commits/41b2888f87fa66eb9e03b73c3107db4287e09ca2